plugins/isplink-client-registration/src/public.php

```diff
@@ -0,0 +1,229 @@
+
+                <!-- Main Card -->
+                <div class="card mb-4">
+                    <div class="card-header">
+                        <h5 class="card-title">Service Address Check</h5>
+                    </div>
+                    <div class="card-body">
+                        <p class="text-muted mb-4">Please enter your service address below to check if our high-speed internet service is available at your location.</p>
+                        
+                        <!-- Address Check Form -->
+                        <form method="POST" data-controller="accounts--status-address-search">
+                            <div class="mb-3">
+                                <label for="street_address" class="form-label">Service Address</label>
+                                <input type="text" 
+                                       class="form-control" 
+                                       id="street_address"
+                                       name="street_address" 
+                                       placeholder="Enter your complete address"  
+                                       autocomplete="off"
+                                       data-target="accounts--status-address-search.addressInput"
+                                       required>
+                                <input type="hidden" 
+                                       name="address_id" 
+                                       data-target="accounts--status-address-search.addressIdInput">
+                                       
+                                <div class="d-flex justify-content-between align-items-center mt-2">
+                                    <span class="text-left loading-matches invisible" 
+                                          data-target="accounts--status-address-search.loadingMessage">
+                                        <small>Loading...</small>
+                                    </span>
+                                    <span class="text-right unknown-address invisible" 
+                                          data-target="accounts--status-address-search.notInSystemMessage">
+                                        <small>Address not found in system</small>
+                                    </span>
+                                </div>
+                            </div>
+                            <div class="d-grid gap-2 mt-4">
+                                <button type="submit" class="btn btn-primary">
+                                    Check Availability
+                                </button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+
+                <!-- Help Section -->
+                <div class="text-center mt-4">
+                    <p class="text-muted">
+                        Need help? <a href="https://isplink.app/" class="text-decoration-none">Contact our support team</a>
```

Comment:
Could you let me know if you are sure about this?

Reply:
@janprochazkacz - Yes, thanks for checking - this link will be available in the "preview" mode of the plugin before the API Key has been provided.   It is to provide support to someone who is previewing the plugin and would like to get it installed.   Once they have entered an API key the link will not be shown.